src/test/java/stringcalculator/StringParser.java

```diff
@@ -0,0 +1,29 @@
+package stringcalculator;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringParser {
+    private static final String DELIMITER = ",|:";
+    private static final Pattern CUSTOM_PATTERN = Pattern.compile("//(.)\n(.*)");
+    private static final int SEPARATOR_GROUP_NUMBER = 1;
+    private static final int EXPRESSION_GROUP_NUMBER = 2;
+
+    public static int[] parser(String s) {
+        Matcher matcher = CUSTOM_PATTERN.matcher(s);
+        if (matcher.find()) {
+            return split(matcher.group(EXPRESSION_GROUP_NUMBER), String.join("|", DELIMITER, matcher.group(SEPARATOR_GROUP_NUMBER)));
```

Comment:
![image](https://user-images.githubusercontent.com/45464819/128353488-8c4f905e-d1fc-47a8-9269-68c71a1f1969.png)
한 라인이 길어지면 읽기 어려울 수 있으니 인텔리제이에서 우측에 선을 넘지 않도록 코드를 작성해주세요ㅎㅎ
또 요런 부분은 한번 끊고 가면 더 가독성이 높아질 수 있겠네요 ^_^
```suggestion
            String delimiter = String.join("|", DELIMITER, matcher.group(SEPARATOR_GROUP_NUMBER));
            return split(matcher.group(EXPRESSION_GROUP_NUMBER), delimiter);
```

Reply:
수정했습니다!